examples/documentation/flake.nix

```diff
@@ -0,0 +1,26 @@
+{
+  description = "Example project demonstrating documentation capabilities.";
+
+  inputs = {
+    pkgs.url = github:NixOS/nixpkgs/nixos-22.11;
+  };
+
+  outputs = { pkgs, ... }:
+    let
+      # TODO: not necessarily
```

Comment:
Same here, either use eachDefaultSystem or note that the example is only for x86_64-linux